quay-enterprise/clair.md

```diff
@@ -0,0 +1,274 @@
+# Clair Setup
+
+<img src="img/Clair_horizontal_color.png" class="img-center" alt="Clair Security Scanner"/>
+
+<a href="https://github.com/coreos/clair">Clair</a> is an open source project for the static analysis
+of vulnerabilities in containers images. It can be used as the engine powering [Quay Security Scanner](security-scanning.md)
+to detect vulnerabilities in all images within Quay, as well as notifiying about new vulnerabilities.
+
+## Initial Setup
+
+### Postgres Database
+
+In order to run Clair, a Postgres database is required. It is recommended to have a Postgres database running
```

Comment:
"For production deployments, we recommend a PostgreSQL database running on machines other than those running Quay, and ideally with automatic replication and failover."